src/main/steps/applicant/yourdocuments/alldocuments/respond_others_allegation_of_harm_and_violence/content.ts

```diff
@@ -0,0 +1,52 @@
+import { TranslationFn } from '../../../../../app/controller/GetController';
+import { FormContent } from '../../../../../app/form/Form';
+
+const en = () => {
+  return {
+    section: 'All documents',
+    title: "Your response to other's allegation of harm and violence",
+    threeHint: 'This is a 8 character code',
+    summaryText: 'Contacts for help',
+    caseNumber: 'Case number',
+    continue: 'Go back',
+  };
+};
+
+const cy: typeof en = () => {
+  return {
```

Comment:
why are the language tag identifier missing?

Reply:
Added welsh tags.